docs/mobility/slam/cartographer/_index.md

```diff
@@ -42,6 +35,15 @@ Cartographer can operate:
 
 ## Use a live machine
 
+{{% alert title="Temporarily not available in live mode" color="caution" %}}
+
+Running `cartographer` in live mode and in the cloud is temporarily disabled.
+The attribute `use_cloud_slam` defaults to `false`.
```

Comment:
@JohnN193 can delete this and below line if you think not necessary to specify?

Reply:
I think it is fine to keep these lines since they reference an attribute listed in the table